syntax = "proto2";

package pkgHeader;

message Header {
  required int32  msgcode = 1;
}

